src/richie/apps/search/filter_definitions/courses.py

```diff
@@ -368,8 +385,16 @@ def aggs_include(self):
         """
         if self.reverse_id:
             if self.base_page:
-                node = self.base_page.node
-                return f".*-{node.path:s}.{{{node.steplen:d}}}"
+                if not self._aggs_include:
+                    self._aggs_include = []
+                    for page in self.base_page.get_child_pages().select_related("node"):
```

Comment:
Wondering how it scales if the number of children is high? Should we limit to a max number of children? For performance but above all for Ux reasons because we can't propose more than a few and should show the "see more" button if there are more results?

Reply:
But at this point the issue is we have not made the ES search request yet, so we do not know which of the children will be in the top 10 values returned by ES.

If there are 10000 children we need to allowlist them all in the `include` param, or look into a way to filter out facets after the ES request is done

Reply:
oh yes! Maybe it scales enough then as long as we cache it good.